scripts/install/RegisterItemRelationsService.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2025 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoAdvancedSearch\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\tao\model\resources\relation\service\ResourceRelationServiceProxy;
+use oat\tao\model\search\SearchProxy;
+use oat\taoAdvancedSearch\model\Resource\Service\ItemRelationsService;
+
+class RegisterItemRelationsService extends InstallAction
```

Comment:
@bartlomiejmarszal can you please move this logic to the existing class `RegisterServices`? It is not necessary to create a new class every time we want to register a new service. 

Thank you! :)

Reply:
Hi @bartlomiejmarszal can you please check my comment above? 

Thanks!